.gitignore

```diff
@@ -1,4 +1,3 @@
 vendor
-composer.lock
```

Comment:
[See this slide from Jordi Boggiano](http://slides.seld.be/?file=2015-06-10+Composer+Best+Practices.html#44) :)